blocks/drawing/package.json

```diff
@@ -24,15 +24,15 @@
   "dependencies": {
     "@blockprotocol/graph": "0.0.18",
     "@tldraw/tldraw": "1.16.0",
-    "@types/react-resizable": "1.7.4",
+    "mobx": "6.5.0",
```

Comment:
Used in `mobx-react-lite`. Block builds fails if this dependency is removed. I moved `mobx` to dependency because it's used in the block itself rather than just during dev.

```
Module not found: Error: Can't resolve 'mobx' in '/tmp/tmp-5458-Bz07RREWURyu/codeload.github.comhashintelhashtar.gzd067f2fec5013cfac4b385a274977bb532263849/node_modules/mobx-react-lite/es'
```